docs/BreakingChanges/084 ICommand CanExecuteChanged behaviour change in .NET 4.5.md

```diff
@@ -0,0 +1,25 @@
+## 84: ICommand CanExecuteChanged behaviour change in .NET 4.5
+
+### Scope
+Minor
+
+### Version Introduced
+4.5
+
+### Version Reverted
+4.5
+
+### Change Description
+In the .NET Framework 4.5, a CanExecuteChangedEvent was ignored unless the sender of the event was the same object as the object that raised the event. This bug was fixed in .NET Framework 4.5 servcing updates.
+
+- [ ] Quirked
+- [ ] Build-time break
+- [ ] Source analyzer planned
```

Comment:
If this can be detected by looking for CanExecuteChanged usage, we may want to plan an analyzer - even if the analyzer is a simple compilation-level check for whether or not that API is used.